api/agent/call.go

```diff
@@ -94,16 +94,17 @@ func FromRequest(appName, path string, req *http.Request) CallOpt {
 			Type:   route.Type,
 			Format: route.Format,
 			// Payload: TODO,
-			Priority:    new(int32), // TODO this is crucial, apparently
-			Timeout:     route.Timeout,
-			IdleTimeout: route.IdleTimeout,
-			Memory:      route.Memory,
-			CPUs:        route.CPUs,
-			Config:      buildConfig(app, route),
-			Headers:     req.Header,
-			CreatedAt:   strfmt.DateTime(time.Now()),
-			URL:         reqURL(req),
-			Method:      req.Method,
+			Priority:       new(int32), // TODO this is crucial, apparently
+			Timeout:        route.Timeout,
+			IdleTimeout:    route.IdleTimeout,
+			Memory:         route.Memory,
+			CPUs:           route.CPUs,
+			FilesystemSize: 0, // TODO make it configurable per route as well
```

Comment:
Need to implement this TODO before merging, at the moment the min returns 0 because of this.

Reply:
I think a system (fleet) wide setting is fine for this, no? To my understanding, this doesn't have to be large due to overlay layers, right?

Reply:
Yeah, now I fixed the logic to determine the minimum limits later on, so we can leave this at 0 for now and defer the problem of whether to support per-route disk space limits.